followthemoney/schema/Position.yaml

```diff
@@ -0,0 +1,65 @@
+Position:
+  label: "Position"
+  plural: "Positions"
+  matchable: false
+  # cf. https://www.popoloproject.com/specs/post.html
+  description: >
+    A post, role or position within an organization or body.
+    This describes a position one or more people may occupy
+    and not the occupation of the post by a specific individual at a 
+    specific point in time.
+
+    'subnationalArea' should be used to further restrict the scope of the
+    position. It should not simply represent some regional aspect of the
+    role - e.g. the constituency of a national member of parliament - when
+    their their legislative jurisdiction is nationwide.
```

Comment:
I'm not sure if I'm overthinking this. My gut tells me it's helpful to decide whether this field is restrictive (the scope of the position is focused on this area) or additive (hey this area is interesting to this position). Happy to delete if it's not helpful now.

Reply:
To me, the use case of this is to define subnational levels of government, so I think the description fits extremely well.